libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-details.component.ts

```diff
@@ -63,10 +63,6 @@ import { DhActualAddressComponent } from './dh-actual-address.component';
   `,
   template: `
     <watt-card *transloco="let t; read: 'meteringPoint.overview.details'">
-      <watt-card-title>
-        <h3>{{ t('title') }}</h3>
```

Comment:
suggestion: let's remove the translation if it's not used anywhere else

Reply:
Removed